AdyenActions/AdyenActionComponent.swift

```diff
@@ -34,6 +34,9 @@ public final class AdyenActionComponent: ActionComponent, Localizable {
     /// :nodoc:
     public var currentActionComponent: Component?
     
+    /// :nodoc:
+    public var threeDSRequestorAppURL: URL?
```

Comment:
I am suggesting create `AdyenActionComponentConfiguration` and move all those fields there instead of setting it here.

Reply:
Also we should add docs to all  public fields in this class here.